lib/libc/riscv/SYS.h

```diff
@@ -47,17 +47,15 @@
 	clgc CAPABILITY_REG(outreg), _C_LABEL(function)
 #define _GET_LOCAL_FNPTR(outreg, function)	\
 	cllc CAPABILITY_REG(outreg), _C_LABEL(function)
-#define _CALL_FNPTR(reg)	cjalr CAPABILITY_REG(reg)
+#define _CALL_FNPTR(reg)	cjalr cra, CAPABILITY_REG(reg)
```

Comment:
Shouldn't be needed now?

Reply:
Yeah I made that change before modifying the compiler. Will revert.

Reply:
```suggestion
#define _CALL_FNPTR(reg)	cjalr CAPABILITY_REG(reg)
```